installation/docker/docker-compose.yml

```diff
@@ -39,6 +39,10 @@ services:
       replicas: 1
       restart_policy:
         condition: on-failure
+    volumes:
+      - type: bind
+        source: /var/run/docker.sock
+        target: /var/run/docker.sock
```

Comment:
We'll probably need to remove this from the e2e compose overrides if we add it here, they also mount /tmp as part of that so we may also require that volume mount.

Reply:
https://github.com/openclarity/vmclarity/blob/main/e2e/docker-compose.override.yml#L4-L10

Reply:
Yeah I saw this, do you know the reason we mount /tmp?

Reply:
@Tehsmash @chrisgacsal any idea why the /tmp mount is needed?

Reply:
@paralta Do you happen to remember why we need `/tmp` to be mounted? Is it a E2E specific requirement?

Reply:
@fishkerez I'd just mvoe the `docker.sock` to the main docker-compose file and leave the `/tmp` in the E2E override file for now. We will revisit the need for `/tmp` separately if that is ok with everyone.